src/VisualStudio/CSharp/Impl/EventHookup/EventHookupSessionManager.cs

```diff
@@ -35,6 +43,25 @@ internal void EventHookupFoundInSession(EventHookupSession analyzedSession)
                 caretPoint.HasValue &&
                 analyzedSession.TrackingSpan.GetSpan(CurrentSession.TextView.TextSnapshot).Contains(caretPoint.Value))
             {
+                // tooltips text is: Program_MyEvents;      (Press TAB to insert)
+                // GetEventNameTask() gets back the event name, only needs to add a semicolon after it.
```

Comment:
Why isn't this being done inside GetEventNameTask?

Reply:
the GetEventNameTask() is also used by the TabKey command handler which adds the actually event handler into the document